manifests/profile/unison.pp

```diff
@@ -12,6 +12,17 @@
   Array $servers = [],
   Array $clients = []
 ) {
+  include nebula::profile::logrotate
+
+  logrotate::rule { 'unison':
+    path          => '/var/log/unison.log',
```

Comment:
`/var/log/unison.log` won't be sufficient; the client logs to `/var/log/unison_<%= @title %>.log`. I believe wildcards can be specified in the logrotate config (that's what we're currently doing for apache), so I think just changing it to `/var/log/unison*.log` should be fine.

Reply:
I've done exactly that, so this should be ready to merge.